INDEX.md

```diff
@@ -0,0 +1,16 @@
+Aktualizr is the codename of the HERE OTA Connect client. This client runs on any embedded device and can check for updates periodically or triggered by another system interaction. You can build this client from source or you can integrate it into your firmware with the Yocto toolset.
```

Comment:
"Codename" feels like a funny way to describe aktualizr. I'd prefer to just say "Aktualizr is the HERE OTA Connect client."

Reply:
OK, reworded.